docs/UserGuide.md

```diff
@@ -31,104 +31,158 @@
+> * For floating tasks, none of the TIMEDATE values are specified.<br>
+> * If priority is not specified, default priority of * is set.
+> * If START_TIMEDATE exceeds END_TIMEDATE, the addition of task will not be successful.
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add Buy eggs at 5pm 13/09/2016 ***`
+* `add Meeting from 13/09/2016 5pm to 13/09/2016 7pm **`
+* `add Pay bills by 13/09/2016 5pm`
+* `add Do laundry *`
+
+#### Displaying tasks : `show`
+Displays tasks and their indexes in the specified timeframe.<br>
+Format: `show [TYPE]`
+
+> * TYPE format: complete, all, overdue, floating, today, tomorrow, upcoming, or DATE.
+> * Except for show complete, all other inputs for TYPE will only display the incomplete tasks, i.e., the tasks which have not been marked as done
+
+Examples:
+* `show floating`<br>
+
+> * Lists all the floating tasks
+
+* `show tomorrow`<br>
+
+> * Lists all tasks for tomorrow
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+* `show overdue`<br>
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> * Lists all the overdue tasks
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+#### Searching for tasks : `find`
+Lists tasks whose names match the given input.<br>
+Format: `find SEARCHSTRING`
+
+> * The search is case insensitive. e.g `buy` will match `Buy`
+> * Wild cards can be indicated with the asterisk `*` e.g. `B*s` will match `Buy eggs`
 > * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * Only full words will be matched e.g. `Buy` will not match `Buys`
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find b*y`<br>
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+> * Returns `buy` but not `buy eggs`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+* `find b*y*`<br>
 
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+> * Returns `buy eggs` but not `must buy eggs`
```

Comment:
This does not meet common habit of users. Users usually expect that `find b*y*` can return `must buy eggs`. 
Don’t design the command format solely to make parsing easier. Primarily, it should be easy to remember and use. Ease of parsing is secondary